lib/diplomat/rest_client.rb

```diff
@@ -256,7 +256,9 @@ def send_get_request(connection, url, options, custom_params = nil)
           rest_options[:headers].map { |k, v| req.headers[k.to_sym] = v } unless rest_options[:headers].nil?
           req.options.timeout = options[:timeout] if options[:timeout]
         end
-      rescue Faraday::ClientError, Faraday::ServerError => e
+      rescue Faraday::TimeoutError => e
+        raise e
+      rescue *faraday_error_classes => e
```

Comment:
This part of existing code sucks because it raises a Diplomat::PathNotFound in most cases, but users might rely on it... just rescue `Faraday::Error` and don't catch `Faraday::TimeoutError`